cmd/exo/cmd/vm_update_nic_ip.go

```diff
@@ -0,0 +1,72 @@
+package cmd
+
+import (
+	"errors"
+	"fmt"
+	"net"
+	"os"
+	"text/tabwriter"
+
+	"github.com/exoscale/egoscale"
+	"github.com/exoscale/egoscale/cmd/exo/table"
+
+	"github.com/spf13/cobra"
+)
+
+// vmUpdateNicIP represents the updateVmNicIp command
+var vmUpdateNicIP = &cobra.Command{
+	Use:   "updatenicip <nic_id> <nic_ip>",
```

Comment:
I don't think any UUID serves as a good UI.

```
% exo vm updateip <privnet> <ip>
```

privnet could be either by name or id.

Then, as always, we have to address the symmetry/consistency of the cli. How does this fit into the other calls between instances and networks?

Reply:
I also need the VM name or ID, the privnet is not enough (I need the nic id). It could be `exo vm updateip <vm> <privnet> <ip>`

Reply:
(right)